digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -574,7 +577,11 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
 
   private def setProjectDeltaToDB(projectDelta:Delta, projectId:Long):Boolean= {
     RoadAddressChangesDAO.clearRoadChangeTable(projectId)
-    RoadAddressChangesDAO.insertDeltaToRoadChangeTable(projectDelta, projectId)
+    val batchInsertionResult = RoadAddressChangesDAO.insertDeltaToRoadChangeTable(projectDelta, projectId)
```

Comment:
to earlier comment, if we had newroad seq in projectdelta case class we could just call RoadAddressChangesDAO.insertDeltaToRoadChangeTable(projectDelta, projectId), instead of looping sqlu. Just have to modify RoadAddressChangesDAO.insertDeltaToRoadChangeTable to handle newroads.